packages/schemas/alterations/next-1709086085 -user-password-encrypt-method.ts

```diff
@@ -0,0 +1,27 @@
+import { sql } from 'slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    await pool.query(sql`
+      alter type users_password_encryption_method add value 'sha1';
+      alter type users_password_encryption_method add value 'sha256';
+      alter type users_password_encryption_method add value 'md5';
+    `);
+  },
+  down: async (pool) => {
+    await pool.query(sql`
+      create type users_password_encryption_method_revised as enum ('Argon2i');
+
+      alter table users 
+      alter column password_encryption_method type users_password_encryption_method_revised 
+      using password_encryption_method::text::users_password_encryption_method_revised;
```

Comment:
This will be a breaking change if the current password_encryption_method is `sha256` ?

Reply:
updated